tools/cockpit.spec

```diff
@@ -413,10 +414,10 @@ Provides: cockpit-shell = %{version}-%{release}
 Provides: cockpit-systemd = %{version}-%{release}
 Provides: cockpit-tuned = %{version}-%{release}
 Provides: cockpit-users = %{version}-%{release}
-Obsoletes: cockpit-dashboard
+Obsoletes: cockpit-dashboard < %{version}-%{release}
 %if 0%{?rhel}
 Provides: cockpit-networkmanager = %{version}-%{release}
-Obsoletes: cockpit-networkmanager
+Obsoletes: cockpit-networkmanager < %{version}-%{release}
```

Comment:
I actually wonder if this condition makes sense/is needed. This was changed in https://github.com/cockpit-project/cockpit/commit/3a3b95deeadcf6612c04252f055124e8026b4f7b for centos-7 which we don't support anymore (from master) and now cannot think of use case where this would be needed.

Reply:
You tell me :-)

Reply:
@martinpitt Do you see a reason to keep this Obsoletes?

Reply:
I agree we can drop it entirely now.